pages/tokens.py

```diff
@@ -84,10 +87,21 @@
                 sm=6,
             ),
         ]),
-
         html.Div(id='token-table'),
-
+        #Initializes a Dash callback store named 'selected-rows' with initial data as an empty list
+        #The store is updated by Dash callbacks based on user interaction (when user makes row selection)
+        dcc.Store(id='selected-rows', data=[]),
         html.Br(),
+        #Create a confirmation dialog provider with a button(Delete Selected tokens) triggering a confirmation message.
+        #Allow users to delete tokens with users confirmation.
+        dcc.ConfirmDialogProvider(
+            children=html.Button('Delete Selected Tokens', id = 'delete-button', disabled = not (os.getenv('TOKEN_DELETION_ENABLED', 'False').lower() == 'true'), style = {'float': 'left',  
+            'font-size': '14px', 'width': '160px', 'display': 'block', 'margin-bottom': '10px',
+            'margin-right': '5px', 'height':'40px', 'verticalAlign': 'top', 'background-color': 'green',
+            'color': 'white',}),
+            id='confirm-delete',
+            message='Are you sure you want to delete selected token(s)?',
+        ),
```

Comment:
Can we add styles when the button is disabled? For example, change the background color to grey when it is disabled and change the background color to green when it is enabled. When I tested it, I got confused because I thought the button was clickable, but it was not.

Also, for testing purposes, I set `TOKEN_DELETION_ENABLED` to `True`, but the button was still not clickable. Is there any other setting I need to change to enable the delete function?

Reply:
Yes, that's a great idea. Styling the active and disabled button differently easily help user differentiate between the two states.

You also need to make this true
``` token_deletion_enabled = os.getenv('TOKEN_DELETION_ENABLED', 'False').lower() == 'true'```

Reply:
@achasmita @JGreenlee I don't understand this comment

> You also need to make this true `os.getenv('TOKEN_DELETION_ENABLED', 'False').lower()`

If @jiji14 sets `TOKEN_DELETION_ENABLED` to True, then `os.getenv` will return `True` instead of the default `False` so it should still work, right?

Reply:
@jiji14 have you done this styling yet?